index.html

```diff
@@ -9,7 +9,24 @@
     <!-- Шапка сайта -->
     <header>Шапка сайта</header>
     <!-- Секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
+			<div class="wrapper"></div>
+				<div class="text-block">
+					<h1 class="title">Peaceful shopping mindful money</h1>
+					<p class="copy">See how we can help with making your shopping experience and money management more ZEN.</p>
+					<div class="links">
+						<a class="link" href="https://apple.com">
+							<img src="./assets/icons/btn-apple.svg" alt="Apple icon">
+						</a>
+						<a class="link" href="htpps://google.com">
+							<img src="./assets/icons/btn-google.svg" alt="Google icon">
+						</a>
+					</div>
+					</div>
+				<div class="image-block">
+					<img src="./assets/images/1.jpg" alt="Bag and device">
+				</div>
+		</section>
```

Comment:
Use BEM methodology
Use "download" class name
Use "download" id name
Use a tab of 2 spaces